_includes/video/pb-os-nas-renderer.html

```diff
@@ -2,11 +2,10 @@
 <html lang="en" class="html--no-js">
     <head>
         {% include head--common.html %}
-        <!--production version of prebid.js-->
-        <script async src="//cdn.jsdelivr.net/npm/prebid.js@latest/dist/not-for-prod/prebid.js"></script>
+        {% include prebidjs-non-prod.html %}
 
         <!--pbjs and player code -->
-        <script type="text/plain" class="optanon-category-C0003">
+        <script >
```

Comment:
not clear why this isn't checking for CMP approval -- outstream renderers could set cookies like in-page players, right?

Reply:
This code does nothing as long the relevant js is loaded (prebid.js and player).
Only callbacks to a que are pushed